src/controllers/users.js

```diff
@@ -21,21 +71,45 @@ exports.post = (request, response, next) => {
           if (!res.rows[0]) {
             checkId(idNumber).then((result) => {
               if (!result.rows[0]) {
-                addUser(data, hash, role)
-                  .then(() => {
-                    response.send({ message: 'User has been added successsfully' });
-                  }).catch((err) => { next(err); });
+                checkMobile(mobile).then((resul) => {
```

Comment:
can you change `resul` to `mobileResult`? We need to keep our code as clean as possible.

Reply:
Done